src/init.ts

```diff
@@ -610,26 +610,6 @@ export class NightwatchInit {
     }
     Logger.info();
 
-    const driversDownloadedFromNPM: { [key: string]: string } = {
-      geckodriver: 'Firefox',
-      chromedriver: 'Chrome'
-    };
-
-    for (const webdriver of webdriversToInstall) {
-      if (webdriver in driversDownloadedFromNPM) {
-        Logger.info(`Installing webdriver for ${driversDownloadedFromNPM[webdriver]} (${webdriver})...`);
-        try {
-          execSync(`npm install ${webdriver} --save-dev`, {
-            stdio: ['inherit', 'pipe', 'inherit'],
-            cwd: this.rootDir
-          });
-          Logger.info(colors.green('Done!'), '\n');
-        } catch (err) {
-          Logger.error(`Failed to install ${webdriver}. Please run 'npm install ${webdriver} --save-dev' later.\n`);
-        }
-      }
-    }
-
```

Comment:
Should we remove this part too: 
https://github.com/nightwatchjs/create-nightwatch/pull/93/files#diff-0e586e6c3496396d18792bb5dfc86ecfb5dd0fd43f81f33e55672bb2bee2ae79L973-L979

Reply:
If Edge is automatically downloaded using selenium-manager, then sure!

Reply:
Also, we should modify the `identifyWebdriverToInstall` method as well to not add the webdrivers in the `webdriversToInstall` array.

Reply:
I tested on my system, Edge driver seems to work fine with selenium manager.